src/monster.cpp

```diff
@@ -660,11 +660,11 @@ static std::pair<std::string, nc_color> speed_description( float mon_speed_ratin
     const std::array<std::tuple<float, nc_color, std::string>, 8> cases = {{
             std::make_tuple( 1.30f, c_red, _( "It is much faster than you." ) ),
             std::make_tuple( 1.00f, c_yellow, _( "It is faster than you." ) ),
-            std::make_tuple( 0.70f, c_white_yellow, _( "It is a bit faster than you." ) ),
+            std::make_tuple( 0.70f, c_brown, _( "It is a bit faster than you." ) ),
```

Comment:
brown is quite dark compared to the other colors, though I don't know of any other options.

Reply:
I thought about that too. We can remove the brown and move it up, making `much faster` with a red background to get the most attention.

Reply:
<img width="230" alt="4" src="https://user-images.githubusercontent.com/38557723/107094160-9a77a180-6817-11eb-8a84-ca7400a2164b.png">

Reply:
I didn't think I would be a big fan of that, but you know what, I think it is an improvement.

Reply:
Ok. I will probably wait for more opinions. And if nothing changes, I'll commit without brown.

Reply:
Green is also rather dark, at least to me; it might not be on a properly-adjusted terminal window, however, and I'm not sure what to do about it in any event.

Reply:
What about using yellow > light red > red for the faster speeds?

Reply:
> What about using yellow > light red > red for the faster speeds?

Great idea! I forgot about it. But it is used to indicate incomplete hp between red and yellow! And no need colored background!

<img width="225" alt="6" src="https://user-images.githubusercontent.com/38557723/107112952-64184180-686c-11eb-9456-0d700a2a3baf.png">

Reply:
> Green is also rather dark, at least to me; it might not be on a properly-adjusted terminal window, however, and I'm not sure what to do about it in any event.

I don't know about the terminal, but green is used throughout the game, so it's pretty common. This is probably the most common colored text (when you are doing well). It used for morale buffs, hp strips, a list of loaded files when the game loading, etc. Maybe I really have different color settings in the console, but this is the same color that everyone is used to :)
Now with light red all colors are common.

Reply:
It doesn't look the same to me - probably indeed different color settings between mine and the pictures.